tests/flag_util_test.py

```diff
@@ -132,5 +133,18 @@ def testReadAndWrite(self):
     self.assertFlagState(flag_values_copy, 2, True)
 
 
+class TestUnitsParser(unittest.TestCase):
+  def setUp(self):
+    self.up = flag_util.UnitsParser()
```

Comment:
No; but since you are defining the "B" type, it might be useful to add a few covering ilo and ibibytes with varying prefixes.